src/app/src/main/kotlin/org/vaccineimpact/api/app/repositories/jooq/JooqUserRepository.kt

```diff
@@ -5,15 +5,15 @@ import org.vaccineimpact.api.app.repositories.UserRepository
 import org.vaccineimpact.api.db.Tables.*
 import org.vaccineimpact.api.db.fromJoinPath
 import org.vaccineimpact.api.db.tables.records.AppUserRecord
-import org.vaccineimpact.api.models.permissions.ReifiedRole
 import org.vaccineimpact.api.models.Scope
-import org.vaccineimpact.api.models.permissions.User
-import org.vaccineimpact.api.models.permissions.UserProperties
-import org.vaccineimpact.api.models.permissions.ReifiedPermission
+import org.vaccineimpact.api.models.permissions.*
+
+import org.vaccineimpact.api.security.UserProperties
 
 class JooqUserRepository : JooqRepository(), UserRepository
 {
-    override fun getUserByEmail(email: String): User? {
+    override fun getUserByEmail(email: String): org.vaccineimpact.api.security.MontaguUser?
```

Comment:
Also in this file it would be good to switch back to importing `MontaguUser`, rather than using the fully qualified name.